source/connections.txt

```diff
@@ -0,0 +1,82 @@
+.. _connections-tab:
+
+===========
+Connections
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+A connection is a MongoDB deployment that you can interact with through MongoDB
+Compass. 
+
+.. note::
+
+   .. include:: /includes/multiple-connections.rst
+
+Connections Sidebar
+-------------------
+
+The :guilabel:`Connections` sidebar on the left of the Compass window
+contains the following:
+
+- The  :guilabel:`Active Connections` menu
+- The :guilabel:`Connections` menu
+- a :icon-lg:`Plus` button to add new connections
+
+Active Connections Menu
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When you activate a connection, the connection name appears in the
+:guilabel:`Active Connections` menu at the top of the
+:guilabel:`Connections` sidebar.
+
+If you click the :icon-fa5:`ellipsis-h` button on the right of a
+connection name, a menu with the following items opens:
+
+- View performance metrics
+- Show connection info
+- Copy connection string
+- Favorite 
+- Disconnect
+
+If the connection is already a Favorite connection, the option to Unfavorite
+appears in the menu. 
+
+To see databases and collections within the active connection, click the
+:icon-fa5:`caret-down` on the left of the connection name. 
+
+To switch between active clusters, click on the name of the cluster you
+want to work with in the :guilabel:`Active Connections` menu. 
+
+Connections Menu
+~~~~~~~~~~~~~~~~
+
+The :guilabel:`Connections` menu contains all Favorite and saved
```

Comment:
same comment here as above regarding capitalization of "favorite"

Reply:
Also, it might help to link users to the [Favorite Connections](https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-37461/connect/favorite-connections/) page

Reply:
Done!